pkg/config/config.go

```diff
@@ -736,8 +736,8 @@ func InitConfig(config Config) {
 	// Datadog security agent (runtime)
 	config.BindEnvAndSetDefault("runtime_security_config.enabled", false)
 	config.BindEnvAndSetDefault("runtime_security_config.debug", false)
-	config.BindEnvAndSetDefault("runtime_security_config.policies.dir", "")
-	config.BindEnvAndSetDefault("runtime_security_config.socket", "")
+	config.BindEnvAndSetDefault("runtime_security_config.policies.dir", "/etc/datadog-agent/runtime-security.d")
+	config.BindEnvAndSetDefault("runtime_security_config.socket", "/opt/datadog-agent/run/runtime-security.sock")
```

Comment:
We should probably change to `/var/run/sysprobe/runtime-security.sock` to avoid an extra mount from system probe.

Reply:
where `/var/run/sysprobe/` is specified ? We saw `/opt/datadog-agent/run/sysprobe.sock` here https://github.com/DataDog/datadog-agent/blob/614a67138f54ed88f340897088cb3293570f7798/Dockerfiles/manifests/all-containers/system-probe-configmap.yaml#L13

Reply:
or here : https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config_template.yaml#L869

Reply:
Yep, you are totally right, please keep this like you configured. The `/var/run/sysprobe/runtime-security.sock` is specific to the public helm chart so I will use that default there.